backend/src/main/scala/cromwell/backend/validation/ContinueOnReturnCodeValidation.scala

```diff
@@ -0,0 +1,53 @@
+package cromwell.backend.validation
+
+import cromwell.backend.validation.RuntimeAttributesValidation._
+import cromwell.core._
+import wdl4s.types.{WdlArrayType, WdlStringType}
+import wdl4s.values.{WdlArray, WdlBoolean, WdlInteger, WdlString}
+
+import scalaz.Scalaz._
+
+object ContinueOnReturnCodeValidation {
+  val key = RuntimeAttributesKeys.ContinueOnReturnCodeKey
+
+  lazy val instance = new ContinueOnReturnCodeValidation {}
+
+  lazy val default = instance.withDefault(WdlInteger(0))
+
+  lazy val optional = default.optional
+
+  val missingMessage =
+    s"Expecting $key runtime attribute to be either a Boolean, a String 'true' or 'false', or an Array[Int]"
+}
+
+trait ContinueOnReturnCodeValidation extends RuntimeAttributesValidation[ContinueOnReturnCode] {
```

Comment:
Saw the concept somewhere else and copied it. I didn't need them to be a full class, nor even an abstract, so I left it a trait.